sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -189,7 +189,7 @@ async def list_ownership(
                 fully_qualified_namespace, eventhub_name, consumer_group
             )
             blobs = self._container_client.list_blobs(
-                name_starts_with=blob_prefix.lower(), include=["metadata"], **kwargs
+                name_starts_with=blob_prefix, include=["metadata"], **kwargs
```

Comment:
this looks good to me, @annatisch would we consider this a breaking change? To me it falls under bug fix, but want to double click

Reply:
@iscai-msft this could potentially be a breaking change for users who are using the current name. So, it needs to be tested how things work when something was created using the current version and then switch over to this change.

Reply:
The name should be lower-cased.  Those languages not doing that should be fixed.